### PR TITLE
add tracking paper

### PR DESCRIPTION
**Pull request by Jee-King**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -428,6 +428,15 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
...
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
+- <a name="zhang2021multi"></a>Zhang, J., Zhao, K., Dong, B., Fu, Y., Wang, Y., Yang, X., and Yin, B.,  
+*[Multi-domain collaborative feature representation for robust visual object tracking](https://doi.org/10.1007/s00371-021-02237-9)*,  
+The Visual Computer, 2021. [PDF](https://link.springer.com/article/10.1007/s00371-021-02237-9), [Project](https://zhangjiqing.com/publication/multi-domain-collaborative-feature-representation-for-robust-visual-object-tracking-the-visual-computer-2021-proc-cgi-2021-/).
+- <a name="zhang2021object"></a>Zhang, J., Yang, X., Fu, Y., Wei, X., Yin, B., Dong, B.,  
+*[Object Tracking by Jointly Exploiting Frame and Event Domain](https://openaccess.thecvf.com/content/ICCV2021/papers/Zhang_Object_Tracking_by_Jointly_Exploiting_Frame_and_Event_Domain_ICCV_2021_paper.pdf)*,  
+ICCV, 2021. [Project](https://zhangjiqing.com/publication/iccv21_fe108_tracking/), [PDF](https://arxiv.org/abs/2109.09052), [code](https://github.com/Jee-King/ICCV2021_Event_Frame_Tracking), [dataset](https://zhangjiqing.com/dataset/).
```
Comment (guillermogb) [inline README.md:436]: Please expand ICCV. See other ICCV entries.

File: README.md [hunk 2]
```diff
@@ -428,6 +428,15 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
...
+- <a name="zhang2022stnet"></a>Zhang, J., Dong, B., Zhang, H., Ding, J., Heide, F., Yin, B., Yang, X.,  
+*[Spiking Transformers for Event-based Single Object Tracking](https://openaccess.thecvf.com/content/CVPR2022/papers/Zhang_Spiking_Transformers_for_Event-Based_Single_Object_Tracking_CVPR_2022_paper.pdf)*,  
+CVPR, 2022. [Project](https://zhangjiqing.com/publication/stnet/), [PDF](https://openaccess.thecvf.com/content/CVPR2022/papers/Zhang_Spiking_Transformers_for_Event-Based_Single_Object_Tracking_CVPR_2022_paper.pdf), [code](https://github.com/Jee-King/CVPR2022_STNet).
```
Comment (guillermogb) [inline README.md:439]: Please expand CVPR. See other CVPR entries.

File: README.md [hunk 3]
```diff
@@ -428,6 +428,15 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
 IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021. [Youtube](https://youtu.be/ST6Z-3SlNS4), [Code](https://github.com/anindya2001/GSCEventMOD).
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
+- <a name="zhang2021multi"></a>Zhang, J., Zhao, K., Dong, B., Fu, Y., Wang, Y., Yang, X., and Yin, B.,
```
Comment (guillermogb) [inline README.md:431]: no "and"

File: README.md [hunk 4]
```diff
@@ -428,6 +428,15 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
 IEEE Int. Conf. Computer Vision Workshop (ICCVW), 2021. [Youtube](https://youtu.be/ST6Z-3SlNS4), [Code](https://github.com/anindya2001/GSCEventMOD).
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
+- <a name="zhang2021multi"></a>Zhang, J., Zhao, K., Dong, B., Fu, Y., Wang, Y., Yang, X., and Yin, B.,  
+*[Multi-domain collaborative feature representation for robust visual object tracking](https://doi.org/10.1007/s00371-021-02237-9)*,  
+The Visual Computer, 2021. [PDF](https://link.springer.com/article/10.1007/s00371-021-02237-9), [Project](https://zhangjiqing.com/publication/multi-domain-collaborative-feature-representation-for-robust-visual-object-tracking-the-visual-computer-2021-proc-cgi-2021-/).
+- <a name="zhang2021object"></a>Zhang, J., Yang, X., Fu, Y., Wei, X., Yin, B., Dong, B.,
```
Comment (guillermogb) [inline README.md:434]: For consistency, the key should be Zhang21iccv, unless it is already taken. See other entries.

File: README.md [hunk 5]
```diff
@@ -428,6 +428,15 @@ Int. Conf. on Machine Vision and Applications (ICMVA), 2021. [Code](https://gith
...
+- <a name="zhang2021multi"></a>Zhang, J., Zhao, K., Dong, B., Fu, Y., Wang, Y., Yang, X., and Yin, B.,  
+*[Multi-domain collaborative feature representation for robust visual object tracking](https://doi.org/10.1007/s00371-021-02237-9)*,  
+The Visual Computer, 2021. [PDF](https://link.springer.com/article/10.1007/s00371-021-02237-9), [Project](https://zhangjiqing.com/publication/multi-domain-collaborative-feature-representation-for-robust-visual-object-tracking-the-visual-computer-2021-proc-cgi-2021-/).
+- <a name="zhang2021object"></a>Zhang, J., Yang, X., Fu, Y., Wei, X., Yin, B., Dong, B.,  
+*[Object Tracking by Jointly Exploiting Frame and Event Domain](https://openaccess.thecvf.com/content/ICCV2021/papers/Zhang_Object_Tracking_by_Jointly_Exploiting_Frame_and_Event_Domain_ICCV_2021_paper.pdf)*,  
+ICCV, 2021. [Project](https://zhangjiqing.com/publication/iccv21_fe108_tracking/), [PDF](https://arxiv.org/abs/2109.09052), [code](https://github.com/Jee-King/ICCV2021_Event_Frame_Tracking), [dataset](https://zhangjiqing.com/dataset/).
+- <a name="zhang2022stnet"></a>Zhang, J., Dong, B., Zhang, H., Ding, J., Heide, F., Yin, B., Yang, X.,
```
Comment (guillermogb) [inline README.md:437]: For consistency, the key should be Zhang22cvpr, unless it is already taken. See other entries.

**Review by guillermogb — CHANGES_REQUESTED** [review]
Thanks for the new entries. In general, I think they are not about feature detection and tracking but more about larger object tracking. 
To myself: I am thinking of splitting into two subcategories: One for feature tracking (keypoints, corners, etc.) and another one for object tracking (using bounding boxes, etc.)

**Comment by Jee-King** [comment]
I agree with you that feature tracking is different from visual object tracking. Splitting them can make an explicit instruction.

**Review by guillermogb — APPROVED** [review]

**Pull request merged**